src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -283,6 +283,81 @@ class CreateTableAsStatementSegment(BaseSegment):
     )
 
 
+@redshift_dialect.segment()
+class CreateExternalTableStatementSegment(BaseSegment):
+    """A `CREATE EXTERNAL TABLE` statement.
+
+    As specified in https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_EXTERNAL_TABLE.html
+    """
+
+    type = "create_external_table_statement"
+
+    match_grammar = Sequence(
+        "CREATE",
+        "EXTERNAL",
+        "TABLE",
+        Ref("TableReferenceSegment"),
+        Bracketed(
+            OneOf(
+                # Columns and comment syntax:
+                Delimited(
+                    Sequence(
+                        Ref("ColumnReferenceSegment"),
+                        Ref("DatatypeSegment"),
+                    ),
+                ),
+            )
+        ),
+        OneOf(Ref("PartitionedBySegment"), optional=True),
+        "STORED",
```

Comment:
We're missing this bit from the docs above here:
```
[ { ROW FORMAT DELIMITED row_format |
  ROW FORMAT SERDE 'serde_name' 
  [ WITH SERDEPROPERTIES ( 'property_name' = 'property_value' [, ...] ) ] } ]
```
Would you be interested in adding it in? If not you should add a TODO note to the end of the docstring to let future contributors know the grammar is incomplete.

Reply:
The scope is a bit daunting with the many possible properties and values. I will add a TODO and potentially address in a future PR

Reply:
Yeah there's no pressure to add everything at once and we can build up the grammar incrementally 😄